src/crab/rcp.hpp

```
@@ -162,6 +162,8 @@ struct TypeToNumDomain {
     TypeToNumDomain join_over_types(const Reg& reg,
                                     const std::function<void(TypeToNumDomain&, TypeEncoding)>& transition) const;
 
+    std::vector<TypeEncoding> foreach_type(const Reg& reg) const;
```

Clang (14.0.6)

[warning] function 'foreach_type' should be marked [[nodiscard]]

(modernize-use-nodiscard)